packages/zowe-explorer/__tests__/__unit__/extension.unit.test.ts

```diff
@@ -282,6 +285,10 @@ async function createGlobalMocks() {
         value: globalMocks.mockOnDidExpandElement,
         configurable: true,
     });
+    Object.defineProperty(vscode.languages, "registerCodeLensProvider", {
+        value: globalMocks.mockRegisterCodeLensProvider,
+        configurable: true,
+    });
```

Comment:
Can we define a stub for `vscode.languages.registerCodeLensProvider` in `packages/zowe-explorer/__tests__/__mocks__/vscode.ts`?

You can add it within the `languages` namespace on line 838:

```ts
export function registerCodeLensProvider(selector: any, provider: any): Disposable {
    return new Disposable();
}
```

Then we can make a spy for the `mockRegisterCodeLensProvider` property on line 54 of this file:

```ts
mockRegisterCodeLensProvider: jest.spyOn(vscode.languages, "registerCodeLensProvider")
```

This avoids overwriting the property definition for other test suites, and allows folks to spy on this function from other test suites without running into a TypeError.